c++/src/kj/compat/http-test.c++

```diff
@@ -1720,7 +1720,7 @@ KJ_TEST("WebSocket core protocol") {
     KJ_EXPECT(message.get<WebSocket::Close>().reason == "bored");
   }
 
-  auto serverTask = server->close(4321, "whatever");
+  auto serverTask = server->close(4321, "whatever").then([](uint64_t s) { KJ_EXPECT(s == 10); });
```

Comment:
Is the assertion run?  Naively, I'd expect s to be 8 for the length of "whatever", but maybe I'm missing something.

Reply:
The close() function is adding two extra bytes here: https://github.com/capnproto/capnproto/blob/master/c%2B%2B/src/kj/compat/http.c%2B%2B#L2132

I am not sure what the purpose for this is, I'd be interested to know.

Reply:
Ah, looks like it packs the 16-bit code in the first two bytes.